apis/apps/v1alpha1/configconstraint_types.go

```diff
@@ -289,6 +289,23 @@ type ShellTrigger struct {
 	//
 	// +optional
 	Sync *bool `json:"sync,omitempty"`
+
+	// Specifies whether to reconfigure dynamic parameters individually or in a batch.
+	// - Set to 'True' to execute the reload action in a batch, incorporating all parameter changes.
+	// - Set to 'False' to execute the reload action for each parameter change individually.
+	// The default value is 'False'.
+	//
+	// +optional
+	BatchReload *bool `json:"batchReload,omitempty"`
+
+	// When `batchReload` is set to 'True', this parameter allows for the optional specification
+	// of the batch input format that is passed into the STDIN of the script.
+	// If this parameter is not specified, the default format used for STDIN is as follows:
+	// Each updated parameter generates a line that concatenates the parameter's key and value with a equal sign ('=').
+	// These lines are then sorted by their keys and inserted accordingly.
+	//
+	// +optional
+	BatchInputTemplate string `json:"batchInputTemplate,omitempty"`
```

Comment:
Could pls list some more examples of `BatchInputTemplate` in Go Template. These exampels will be show in `kubectl explain` and peope will learn from the documenation.

Reply:
Fixed in d5abe20ac8a0254c609bdd3ff573125f5fa3b3b1